### PR TITLE
FIX: groupmeans handles empty string before ttest

### PR DESCRIPTION
**Pull request by skathirmani**

### CODE REVIEW

File: autolysis/__init__.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def groupmeans(data, groups, numbers,
     means = {col: bz.into(float, data[col].mean()) for col in numbers}
     # pre-create aggregation expressions (mean, count)
     agg = {number: bz.mean(data[number]) for number in numbers}
-    agg['#'] = bz.count(data)
+    agg['#'] = bz.count(data[group])
     results = []
```
Comment (pratapvardhan) [inline autolysis/__init__.py:174]: `group` is not defined yet?

**Comment by pratapvardhan** [comment]
We need to quickly have tests for groupmeans.

**Comment by skathirmani** [comment]
True @pratapvardhan. Lets plan for that. I forgot that you have moved that line out of for loop.. my bad

**Comment by skathirmani** [comment]
I am closing this pull request.. will give a fresh PR

**Pull request closed without merging**